main.js

```diff
@@ -75,24 +76,83 @@ function createTray() {
 }
 
 function startApp() {
-  const modalPath = `file://${__dirname}/process.html`
+  const modalPath = `file://${__dirname}/process.html` // TODO about:blank?
```

Comment:
In order to get the tests working, I had to load the `process.js` thread from this html file, even though the window is not visible. Happy to look into doing it another way - we can refactor and as long as the tests continue running we should be good!